php/php.latte/test/unit/src/org/netbeans/modules/php/latte/indent/LatteIndenterTest.java

```diff
@@ -48,14 +48,6 @@ public void testRealFile_02() throws Exception {
         indent("testRealFile_02");
     }
 
-    public void testRealFile_03() throws Exception {
```

Comment:
Silly question: are these tests invalid/outdated for some reason ? The content of `testRealFile_03/04` seems to reside in HG, and seems as a copy of https://github.com/obrejla/Pollie/blob/master/OndrejBrejla/Pollie/PollieForm.latte

Reply:
I'll add them after correct way how to include them will be consulted with @matthiasblaesing .

Reply:
Remainig test files were added. They contain link to source and license (New BSD License).